nav2_core/include/nav2_core/local_planner_base.hpp

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
`local_planner_base.hpp` -> `local_planner.hpp`

Reply:
Or what we talked about with the controller vs local planner terminology.

I dont care one way or another, but lets keep it consistent

Reply:
@SteveMacenski What are your thoughts on keeping the naming of the local planner plugins as ```planner```, and naming only the action interface as ```controller```? In ROS1, the dwa was still referred to as ```planner``` so I think we should keep it consistent here.